scripts/createIndex.js

```diff
@@ -17,7 +17,7 @@ co(function * createIndex () {
     // fields not specified below will be 'text' by default
     properties: {
       resource: { type: 'keyword' },
-      challengeId: { type: 'long' },
+      challengeId: { type: 'keyword' },
```

Comment:
This is only for local development - It's irrelevant for deployment to prod / dev.

Since we copied over `master` branch, we do not have the code where we corrected the tests and the test data - so, for now, with the test data that we have, type `keyword` is needed..